testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cluster/AuthenticationSessionFailoverClusterTest.java

```diff
@@ -83,8 +84,17 @@ public void setup() {
                 .build();
 
         userId = ApiUtil.createUserAndResetPasswordWithAdminClient(adminClient.realm("test"), user, "password");
-        getCleanup().addUserId(userId);
+        
+        //TODO: weird behavior when doing Quarkus where only one of the required actions above are persisted, so we workaround by forcing an update to the user with the 2 actions we want
+        UserResource userResource = adminClient.realm(testRealm.getRealm()).users().get(userId);
+        UserRepresentation userRepresentation = userResource.toRepresentation();
+        
+        if (userRepresentation.getRequiredActions().size() < 2) {
+            userRepresentation.setRequiredActions(user.getRequiredActions());
+            userResource.update(userRepresentation);
+        }
 
+        getCleanup().addUserId(userId);
```

Comment:
Shouldn't we rather ignore this test for now and fix the problem properly instead of adding a work around to the test?

Reply:
We could. But I would like to have this test running even though we didn't investigate further yet the reason for that issue. That is why I left that there.